src/envoy/mixer/envoy.conf.template

```diff
@@ -65,6 +120,17 @@
             "url": "tcp://${BACKEND}"
           }
         ]
+      },
+      {
+        "name": "service2",
+        "connect_timeout_ms": 5000,
+        "type": "strict_dns",
+        "lb_type": "round_robin",
+        "hosts": [
+          {
+            "url": "tcp://localhost:9090"
+          }
+        ]
```

Comment:
Finally, I suggest using the JSON schema approach that Envoy uses. It's relatively easy to extend and cuts down a ton of unnecessary validation code.